apis/v1alpha2/validation/httproute_test.go

```diff
@@ -19,43 +19,41 @@ package validation
 import (
 	"testing"
 
+	"k8s.io/apimachinery/pkg/util/validation/field"
 	utilpointer "k8s.io/utils/pointer"
 
 	gatewayv1a2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
+	pkgutils "sigs.k8s.io/gateway-api/pkg/util"
 )
 
 func TestValidateHTTPRoute(t *testing.T) {
 	testService := "test-service"
 	specialService := "special-service"
 	tests := []struct {
-		name     string
-		hRoute   gatewayv1a2.HTTPRoute
-		errCount int
+		name        string
+		hRouteRules []gatewayv1a2.HTTPRouteRule
```

Comment:
Tiny nit - "rules" would likely be a bit simpler here.
```suggestion
		rules []gatewayv1a2.HTTPRouteRule
```

Reply:
👍 dcb955a6c679b58cb4e34886940eeab79154916b